toplevel/coqtop.ml

```diff
@@ -537,10 +578,12 @@ let parse_args arglist =
     |"-with-geoproof" -> Coq_config.with_geoproof := get_bool opt (next ())
     |"-main-channel" -> Spawned.main_channel := get_host_port opt (next())
     |"-control-channel" -> Spawned.control_channel := get_host_port opt (next())
-    |"-vio2vo" -> add_compile false (next ()); Flags.compilation_mode := Flags.Vio2Vo
+    |"-vio2vo" -> add_compile false (next ());
+      Safe_typing.allow_delayed_constants := true;
```

Comment:
Setting the flag should be irrelevant in this case, the code path that tests for it should never be used.

Reply:
Ok, noted, thanks.